guides/source/configuring.md

```diff
@@ -1083,6 +1090,7 @@ text/javascript image/svg+xml application/postscript application/x-shockwave-fla
 - `config.active_support.cache_format_version`: `7.0`
 - `config.action_dispatch.return_only_request_media_type_on_content_type`: `false`
 - `config.action_mailer.smtp_timeout`: `5`
+- `config.active_storage.video_preview_arguments`: `"-vf select=eq(n\,0)+eq(key\,1)+gt(scene\,0.015),loop=loop=-1:size=2,trim=start_frame=1 -frames:v 1 -f image2"`
```

Comment:
Would you mind adding an entry to the "Baseline defaults:" section further below as well?  i.e.:

```md
- `config.active_storage.video_preview_arguments`: `"-y -vframes 1 -f image2"`
```

Reply:
Done :)